src/sentry/api/endpoints/organization_traces.py

```diff
@@ -552,9 +553,10 @@ def get_trace_info(trace: str) -> tuple[str, str, float] | tuple[None, None, Non
 
         for trace in traces:
             info = get_trace_info(trace["trace"])
-            trace["project"] = info[0]
-            trace["name"] = info[1]
-            trace["rootDuration"] = info[2]
+            if info[0] is not None and info[1] is not None:
```

Comment:
do you need to check `info[2]` too?

Reply:
no, that's just a duration. if the first 2 are defined, it's good enough